syz-fuzzer/fuzzer.go

```diff
@@ -285,19 +278,55 @@ func main() {
 		fuzzer.execOpts.Flags |= ipc.FlagEnableCoverageFilter
 	}
 
+	var files map[string][]string
+	globs := getGlobs(target)
+	if len(globs) != 0 {
+		files, err = host.CollectDirsInfo(globs)
+		if err != nil {
+			log.Fatalf("faield to collect files info: %v", err)
+		}
+	}
+
 	log.Logf(0, "starting %v fuzzer processes", *flagProcs)
 	for pid := 0; pid < *flagProcs; pid++ {
 		proc, err := newProc(fuzzer, pid)
 		if err != nil {
 			log.Fatalf("failed to create proc: %v", err)
 		}
+		proc.fuzzer.target.Files = files
 		fuzzer.procs = append(fuzzer.procs, proc)
 		go proc.loop()
 	}
 
 	fuzzer.pollLoop()
 }
 
+func collectMachineInfos() ([]byte, []host.KernelModule) {
+	machineInfo, err := host.CollectMachineInfo()
+	if err != nil {
+		log.Fatalf("failed to collect machine information: %v", err)
+	}
+	modules, err := host.CollectModulesInfo()
```

Comment:
If we do CollectMachineInfo and CollectModulesInfo here, it looks reasonable to do CollectDirsInfo here as well.